examples/cpp/pyperf/PyPerf.cc

```diff
@@ -39,6 +40,11 @@ int main(int argc, char** argv) {
   // Argument parsing helpers
   int pos = 1;
 
+  if (std::setvbuf(stderr, NULL, _IOLBF, 0) != 0) {
```

Comment:
Any reason not to use `setlinebuf`? It's more descriptive.

Reply:
```
/home/jong/git/Granulate/bcc/examples/cpp/pyperf/PyPerf.cc:42:8: error: ‘setlinebuf’ is not a member of ‘std’; did you mean ‘stringbuf’?
```

Reply:
I see. `setlinebuf` is a BSD function and needs `_DEFAULT_SOURCE`. Ok, let's keep it `setvbuf`.